emulator/index.php

```diff
@@ -23,6 +23,26 @@
     <meta charset="utf-8">
     <title>Terminal</title>
     <link rel="stylesheet" href="css/screen.css">
+    <script>
+        var commands = [];
+        var which_command = 0;
+
+        $('#the_input_term').keydown(function (e) {
+            if (e.keyCode === 13) {
+                commands.push($('#the_input_term').val());
```

Comment:
After a certain amount (maybe 128) the oldest entry should be removed to limit the size of the commands array.

Reply:
I think its a good idea :)